app/views/admin/attachments/_attachments.html.erb

```diff
@@ -1,45 +1,37 @@
 <% initialise_script "GOVUK.ReorderableAttachmentsList", el: "ol.existing-attachments" if attachable.can_order_attachments? && attachable.attachments.many? %>
```

Comment:
Is this still needed for the design system page?

Reply:
Well spotted @dnkrj